src/librustc_session/parse.rs

```diff
@@ -118,6 +129,7 @@ pub struct ParseSess {
     pub ambiguous_block_expr_parse: Lock<FxHashMap<Span, Span>>,
     pub injected_crate_name: Once<Symbol>,
     pub gated_spans: GatedSpans,
+    pub symbol_gallery: SymbolGallery,
```

Comment:
Could you add a doc comment to this field explaining its purpose?

Reply:
Sure, do you have a suggestion about what this field should be called?

Reply:
I don't know, seems ok as is.

Reply:
Done